src/Seq.Extensions.Logging/Serilog/Extensions/Logging/SerilogLoggerScope.cs

```diff
@@ -76,7 +49,7 @@ public void EnrichAndCreateScopeItem(LogEvent logEvent, ILogEventPropertyFactory
                     }
 
                     var property = propertyFactory.CreateProperty(key, stateProperty.Value, destructureObject);
-                    logEvent.AddPropertyIfAbsent(property);
+                    logEvent.AddOrUpdateProperty(property);
                 }
```

Comment:
Since scope enumerating goes from the outer to the inner now we need to update property with the same name.